app/components/Atoms/Input/Input.Styled.jsx

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+//import '../../../styles/MediaQueries.scss';
```

Comment:
Remove comments

Reply:
Nice catch, comment removed, thank you very much!